src/components/CreateWine/index.js

```diff
@@ -32,29 +42,53 @@ class CreateWine extends Component {
     }
   };
 
+  onCompleted = () => {
+    Toast.fire({
+      type:"success",
+      title:"Wine Added Successfully"
+    });
+    this.setState({
+      name: "",
+      grapes: [],
+      winery: "",
+      year: undefined,
```

Comment:
Since the default state was changed to use `null` (which is a good idea), this should likely be `null` as well to be consistent.